### PR TITLE
Version 1.0.1 + WatchOS example

### PR DESCRIPTION
**Pull request by cygy**
SDK: some code refactoring, version 1.0.1.
Examples: add WatchOS example.

### CODE REVIEW

File: Examples/OVHAPIWrapper-Example-watchOS/OVHAPIWrapper-Example-watchOS WatchKit Extension/DataController.swift [hunk 1]
```diff
@@ -267,6 +267,14 @@ final class DataController: NSObject, WCSessionDelegate {
         }
     }
     
+    func sessionReachabilityDidChange(session: WCSession) {
+        print("iOS application reachability changed.")
```
Comment (yadutaf) [inline Examples/OVHAPIWrapper-Example-watchOS/OVHAPIWrapper-Example-watchOS WatchKit Extension/DataController.swift]: This is debug only, right ?

File: Examples/OVHAPIWrapper-Example-watchOS/OVHAPIWrapper-Example-watchOS WatchKit Extension/VPSRowController.swift [hunk 2]
```diff
@@ -46,7 +46,7 @@ class VPSRowController: NSObject {
                 
                 var color = UIColor.grayColor()
                 if vps.state == .running {
-                    color = UIColor.greenColor()
+                    color = UIColor(red: 0.0, green: 0.8, blue: 0.2, alpha: 1.0)
```
Comment (yadutaf) [inline Examples/OVHAPIWrapper-Example-watchOS/OVHAPIWrapper-Example-watchOS WatchKit Extension/VPSRowController.swift]: Can you use constants/config ?

File: Examples/OVHAPIWrapper-Example-OSX/README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+This project shows you how to use the Swift OVH Wrapper framework into an OS X project
+
+## Platforms
+
+OS X
+
+## How to run
+
+In order to run this example, you have to copy the file `Examples/OVHAPIWrapper-Example-OSX/OVHAPIWrapper-Example-OSX/Resources/Credentials-Sample.plist` to `Examples/OVHAPIWrapper-Example-OSX/OVHAPIWrapper-Example-OSX/Resources/Credentials.plist` and to set the application key, application secret and consumer key relative to your own application (OVH Europe).
```
Comment (yadutaf) [inline Examples/OVHAPIWrapper-Example-OSX/README.md]: Can you add a link to https://api.ovh.com/createToken/index.cgi?GET=/*&PUT=/*&DELETE=/*&POST=/* to help generate credentials (will request full access to the API)

File: Examples/OVHAPIWrapper-Example-iOS/README.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+This project shows you how to use the Swift OVH Wrapper framework into an iOS project
+
+## Platforms
+
+iOS
+
+## How to run
+
+In order to run this example, you have to copy the file `Examples/OVHAPIWrapper-Example-iOS/OVHAPIWrapper-Example-iOS/Resources/Credentials-Sample.plist` to `Examples/OVHAPIWrapper-Example-iOS/OVHAPIWrapper-Example-iOS/Resources/Credentials.plist` and to set the application key, application secret and consumer key relative to your own application (OVH Europe).
```
Comment (yadutaf) [inline Examples/OVHAPIWrapper-Example-iOS/README.md]: Can you add a link to https://api.ovh.com/createToken/index.cgi?GET=/*&PUT=/*&DELETE=/*&POST=/* to help generate credentials (will request full access to the API)

File: Examples/OVHAPIWrapper-Example-watchOS/README.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+This project shows you how to use the Swift OVH Wrapper framework into a WatchOS/iOS project
+
+## Platforms
+
+WatchOS and iOS
+
+## How to run
+
+In order to run this example, you have to copy the file `Examples/OVHAPIWrapper-Example-watchOS/Resources/Credentials-Sample.plist` to `Examples/OVHAPIWrapper-Example-watchOS/Resources/Credentials.plist` and to set the application key, application secret and consumer key relative to your own application (OVH Europe).
```
Comment (yadutaf) [inline Examples/OVHAPIWrapper-Example-watchOS/README.md]: Can you add a link to https://api.ovh.com/createToken/index.cgi?GET=/*&PUT=/*&DELETE=/*&POST=/* to help generate credentials (will request full access to the API)

**Comment by yadutaf** [comment]
Hi,

Looks like an incredible addition! https://github.com/ovh/swift-ovh/pull/4/commits/63347370fae7b3c7f01f19b3b0d4e9ad26eea95e looks more like additions than refactoring. Could you describe a bit more what is being added ?

I :heart: the Gif examples! Although, I suggest a bit of filtering on the domain names ;)

File: Examples/OVHAPIWrapper-Example-OSX/README.md (hunk 3 above)
Comment (cygy) [inline Examples/OVHAPIWrapper-Example-OSX/README.md]: Yes I'm going to.

File: Examples/OVHAPIWrapper-Example-iOS/README.md (hunk 4 above)
Comment (cygy) [inline Examples/OVHAPIWrapper-Example-iOS/README.md]: Yes I'm going to.

File: Examples/OVHAPIWrapper-Example-watchOS/README.md (hunk 5 above)
Comment (cygy) [inline Examples/OVHAPIWrapper-Example-watchOS/README.md]: Yes I'm going to.

**Comment by cygy** [comment]
The additions are about the example, a lot of files is Xcode configuration related.

File: Examples/OVHAPIWrapper-Example-watchOS/OVHAPIWrapper-Example-watchOS WatchKit Extension/DataController.swift (hunk 1 above)
Comment (cygy) [inline Examples/OVHAPIWrapper-Example-watchOS/OVHAPIWrapper-Example-watchOS WatchKit Extension/DataController.swift]: Yes it is debug only, just in the example project, not present in the wrapper framework.

File: Examples/OVHAPIWrapper-Example-watchOS/OVHAPIWrapper-Example-watchOS WatchKit Extension/VPSRowController.swift (hunk 2 above)
Comment (cygy) [inline Examples/OVHAPIWrapper-Example-watchOS/OVHAPIWrapper-Example-watchOS WatchKit Extension/VPSRowController.swift]: It is in the example project, not in the warpper framework, do we really need this?

Comment (yadutaf) [inline Examples/OVHAPIWrapper-Example-watchOS/OVHAPIWrapper-Example-watchOS WatchKit Extension/VPSRowController.swift]: Nope, just a "nice to have"

**Pull request merged**